docs/install-azure-repos.md

```diff
@@ -143,6 +143,11 @@ To install Frogbot on Azure Repos repositories, follow these steps.
             # Template for the pull request title generated by Frogbot when creating pull requests with fixes.
             # The template can optionally include the ${IMPACTED_PACKAGE} and ${FIX_VERSION} variables.
             # JF_PULL_REQUEST_TITLE_TEMPLATE: "[🐸 Frogbot] Upgrade ${IMPACTED_PACKAGE} to to ${FIX_VERSION}"
+
+            # [Optional, Default: "FALSE"]
+            # If true, Frogbot creates a single pull request with all the fixes.
+            # If false, Frogbot creates a separate pull request for each fix.
```

Comment:
true --> TRUE
false --> FALSE
(unlike in the frogbot-config.yml when the above values should be documented as lower-case, here it should be documented as upper-case, beucase of the way this value is set in the line below. Let's implement the same in all workflows).